File: 12.2023/18.12.2023/solution.js
const solution = () => {
  let k = prompt("Enter any number:", 17);

  if (isNaN(k)) {
    alert("Your input was not a number.");
    return;
  }
  k = parseInt(k);
  const list = [10, 15, 3, 7];

  for (let i = 0; i < list.length; i++) {
    for (let j = i + 1; j < list.length; j++) {
      if (list[i] + list[j] === k) {
        alert(`The sum of ${list[i]} and ${list[j]} is ${k}`);
      }
    }
  }
};

solution();

File: 12.2023/18.12.2023/solution.py
def solution():
  k = input("Enter any number: ")

  if not k.isdigit():
    print("Your input was not a number.")
    return

  k = int(k)
  lst = [10, 15, 3, 7]

  for i in range(len(lst)):
    for j in range(i + 1, len(lst)):
      if lst[i] + lst[j] == k:
        print(f"The sum of {lst[i]} and {lst[j]} is {k}")


solution()